### PR TITLE
Fossasia Projects service box

### LINKED ISSUE #530: Add a menu icon and menu items on top right to other FOSSASIA services
On Google there is a menu icon with different services. Please implement the same in loklak search.

screenshot from 2017-08-12 01-36-54

### PR DESCRIPTION
**Pull request by hemantjadon**
* Add box which contains links to Fossasia Projects

* Further links can be added.

* Currently added loklak, susper, susi, and fossasia links. 

* Suggest what other links can be added

screen shot 2017-08-14 at 3 40 43 pm

https://hemantjadon.github.io/loklak_search/

**Closes #530**

### CODE REVIEW

**Comment by singhpratyush** [comment]
@hemantjadon: Please provide a test link.

**Comment by Achint08** [comment]
@hemantjadon What I think is that we are actually not covering all the projects. 

screen shot 2017-08-15 at 12 35 41 am

**Comment by hemantjadon** [comment]
@vibhcool @djmgit I followed the same way the google do this, that's why I used logo + text.

**Comment by mariobehling** [comment]
* Please match the design of the Google box more closely
screenshot from 2017-08-16 12-11-44

* write it FOSSASIA not Fossasia
* Add eventyay.com
* Add phimp.me

**Comment by mariobehling** [comment]
What is the status here? Not sure, why this is taking so long.

**Comment by hemantjadon** [comment]
@jigyasa-grover @mariobehling @singhpratyush review now 😄

**Comment by vibhcool** [comment]
Issue in css design
screenshot from 2017-08-28 23-37-34

File: src/app/service-box/service-box.component.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+				<div class="col">
+					<div class="img">
+						<img src="assets/images/pslab_64x68.png" width="50" height="50" alt="PSLab">
+					</div>
+					<span>PSLab</span>
+				</div>
+			</a>
+		</div>
+		<div class="row">
+			<a href="https://pslab.fossasia.org/">
```
Comment (vibhcool) [inline src/app/service-box/service-box.component.html]: wrong link

File: src/app/service-box/service-box.component.html [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+		<div class="row">
+			<a href="http://susper.com/">
+				<div class="col">
+					<div class="img">
+						<img src="assets/images/susper_60x16.png" width="60" height="16" alt="Susper">
+					</div>
+					<span>Susper</span>
+				</div>
+			</a>
+			<a href="http://chat.susi.ai/">
```
Comment (vibhcool) [inline src/app/service-box/service-box.component.html]: this link isn't working for my laptop. :sweatsmile: ,but is working on mobile.

**Pull request closed without merging**